### PR TITLE
Updated Walk Tool

### PR DESCRIPTION
**Pull request by spap975**
Changes the old walkTool tool to the new LookAndMove tool.

### CODE REVIEW

File: packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx [hunk 1]
```diff
@@ -2,9 +2,14 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
```
Comment (kckst8) [inline packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx:7]: looks like the precommit script added a duplicate header. Can you remove one of them?

File: packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx [hunk 2]
```diff
@@ -2,9 +2,14 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
+/*---------------------------------------------------------------------------------------------
```
Comment (aruniverse) [inline packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx:5]: Duplicate header

**Review by kckst8 — COMMENTED** [review]
Also will need to run "rush change" to generate a version bump/changelog. I will pull this to test later today

File: packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx [hunk 3]
```diff
@@ -31,75 +37,82 @@ export interface BasicNavigationWidgetProps {
 export function BasicNavigationWidget({
   config,
 }: BasicNavigationWidgetProps): JSX.Element {
-  const getHorizontalToolbarItems = React.useCallback((): CommonToolbarItem[] => {
-    if (config?.hideDefaultHorizontalItems) {
-      return [];
-    }
-    const items: CommonToolbarItem[] = [];
+  const getHorizontalToolbarItems =
```
Comment (aruniverse) [inline packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx:40]: are these changes becasue of the changes you made to formatting?

File: packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx [hunk 4]
```diff
@@ -114,7 +127,16 @@ export function BasicNavigationWidget({
       items.push(
         ToolbarHelper.createToolbarItemFromItemDef(
           10,
-          CoreTools.walkViewCommand
+          new ToolItemDef({
```
Comment (aruniverse) [inline packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx:130]: Looks good, i just searched and had found LookAndMoveTool in \core\frontend\src\tools\ViewTool.ts
Would it be possible to use that, or do you need to redefine the tool item like this?

**Review by aruniverse — COMMENTED** [review]
Missing rush change

File: packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx (hunk 3 above)
Comment (spap975) [reply packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx:40]: yeah i just did a shift alt f and I'm pretty sure this is what changed, should I undo the changes?

File: packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx (hunk 4 above)
Comment (spap975) [reply packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx:130]: Just tried using the tool, surprisingly doesn't fit with the typing of the function even though it extends the Tool class

**Review by aruniverse — APPROVED** [review]

File: packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx [hunk 5]
```diff
@@ -114,7 +122,16 @@ export function BasicNavigationWidget({
...
+            toolId: "View.LookAndMove",
+            iconSpec: "icon-walk",
+            execute: () =>
+              IModelApp.tools.run(
+                "View.LookAndMove",
+                IModelApp.viewManager.selectedView
+              ),
+            labelKey: IModelApp.i18n?.translate("iTwinViewer:tools.walkTool"),
```
Comment (aruniverse) [inline packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx:133]:
```suggestion
            labelKey: "iTwinViewer:tools.walkTool",
```

**Comment by aruniverse** [comment]
@spap975 plz take a look at the failing test, and update accordingly

**Pull request merged**